package.json

```diff
@@ -3,17 +3,19 @@
   "version": "1.0.0",
   "description": "User Authentication & Authorization with JWT",
   "engines": {
-    "node": "18.14.2",
+    "node": ">=18.14.2",
     "npm": "9.5.0"
   },
   "main": "server.js",
   "scripts": {
-    "start:server": "nodemon server.js",
+    "start:server": "nodemon ./app/server.js",
     "lint:stage": "lint-staged",
     "lint:commit": "commitlint --edit",
     "format:prettier": "prettier -w .",
     "lint": "eslint .",
-    "lint:fix": "eslint --fix ."
+    "lint:fix": "eslint --fix .",
```

Comment:
Please make sure ESLint and prettier also cover test folder as well

Reply:
This script worked for `test` folder